python/foglamp/services/core/api/update.py

```diff
@@ -0,0 +1,89 @@
+# -*- coding: utf-8 -*-
```

Comment:
I think module name should be `snap_update.py`

Reply:
No, the name is update.py on purpose.

It's a generic interface.


The specialisation, ie the package type and ho to update, belongs to scripts/tasks/update, a generic name too, which is written by the package manager (whatever it is)

Reply:
Okay